crane_x7_tests/launch/pick_and_place.launch.py

```diff
@@ -0,0 +1,148 @@
+import os
+import sys
+from datetime import datetime
+from time import sleep
+
+import pytest
+import unittest
+import launch_testing
+from launch import LaunchDescription
+from launch.actions import ExecuteProcess, TimerAction
+from launch_ros.actions import Node
+
+from ament_index_python.packages import get_package_share_directory
+from launch.actions import IncludeLaunchDescription
+from launch.launch_description_sources import PythonLaunchDescriptionSource
+
+from launch_ros.substitutions import FindPackageShare
+from launch.substitutions import PathJoinSubstitution
+import launch_testing.markers
+
+import subprocess
+import re
+
+
+@pytest.mark.launch_test
+@launch_testing.markers.keep_alive
+
+def generate_test_description():
+    proc_env = os.environ.copy()
+    proc_env["PYTHONUNBUFFERED"] = "1"
+
+    sim = IncludeLaunchDescription(
+        PythonLaunchDescriptionSource(
+            [
+                PathJoinSubstitution(
+                    [
+                        FindPackageShare("crane_x7_gazebo"),
+                        "launch",
+                        "crane_x7_with_table.launch.py",
+                    ]
+                )
+            ]
+        ),
+    )
+
+    demo = IncludeLaunchDescription(
+        PythonLaunchDescriptionSource(
+            [
+                PathJoinSubstitution(
+                    [
+                        FindPackageShare("crane_x7_examples"),
+                        "launch",
+                        "example.launch.py",
+                    ]
+                )
+            ]
+        ),
+        launch_arguments={
+            "use_sim_time": "true", 
+            "example": "pick_and_place"
+        }.items(),
+    )
+
+    
+    return LaunchDescription(
+        [
+            sim,
+            TimerAction(period=15.0, actions=[demo]),  # Need to optimize period, if under 10 seconds will not run
+            launch_testing.actions.ReadyToTest(),
+        ]
+    ), {"demo": demo, "sim":sim}
+
+
+def get_model_location(model_name):
```

Comment:
Do you remember when you dirst worked on that? I think at the beginning wer were planning to use some service, however it was not working with gazebo classic if I recall. Since we have ignition here, is there a better solution?

If not it could be interesting to put that in some helper library at some point (either a standalone package, or part of the artefacts client)

Reply:
As we discussed yesterday despite this not being the cleaner way to solve the issue, it is less invasive in relation to changes we would make to the code. I agree that a really good idea would be to have an "artefacts_sim_bridge" package that would allow for easy access to gz-classic and ignition variables/models/etc...

Reply:
as discussed, the ideal solution would be using ros params, but we can skip for now